apps/core-lib-dev/src/app/calendar.element.ts

```diff
@@ -3,8 +3,6 @@ import { customElement, state } from 'lit/decorators.js'
 import { html } from '@sebgroup/green-core/scoping'
 
 import '@sebgroup/green-core/components/calendar/index.js'
-import * as CalendarTransitionalStyles from '@sebgroup/green-core/components/calendar/calendar.trans.styles.js'
```

Comment:
Was this intentional?

Reply:
Yes, this is just in `core-lib-dev` in order to test without transitional styles!